experiments/process-kill/process-kill.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2022 LitmusChaos Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package experiment
+
+import (
+	"log"
+	"net/http"
+
+	errorcodes "github.com/litmuschaos/m-agent/internal/m-agent/error-codes"
+	"github.com/litmuschaos/m-agent/internal/m-agent/messages"
+	"github.com/litmuschaos/m-agent/internal/m-agent/upgrader"
+	"github.com/litmuschaos/m-agent/pkg/probes"
+	"github.com/litmuschaos/m-agent/pkg/process-kill"
+)
+
+// ProcessKill listens for the client actions and executes them as appropriate
+func ProcessKill(w http.ResponseWriter, r *http.Request) {
+
+	// upgrade the connection to a websocket connection
+	upgrader := upgrader.GetConnectionUpgrader()
+
+	conn, err := upgrader.Upgrade(w, r, nil)
+	if err != nil {
+		log.Printf("Failed to establish connection with client, %v", err)
```

Comment:
We are not using logrus here; logging is taken care of by the default log pkg. In my opinion, we don't need logrus or Errorf or Infof, as we are not logging any critical experiment info, since all the information or error is sent back to the client directly.